SimGeneral/MixingModule/plugins/MixingModule.cc

```diff
@@ -383,6 +383,8 @@ namespace edm {
     ModuleCallingContext moduleCallingContext(&moduleDescription());
     ModuleContextSentry moduleContextSentry(&moduleCallingContext, parentContext);
 
+    setupPileUpEvent(setup);
```

Comment:
Just to note that as far as I can tell, the necessity of this call has gone unnoticed because before we have not used the ability to run modules (that would consume EventSetup products) as part of the MixingModule. Would you agree?

Reply:
I think you are probably correct.